ts/store/reducers/wallet/wallets.ts

```diff
@@ -199,6 +214,21 @@ export const creditCardListSelector = createSelector(
     )
 );
 
+/**
+ * Return a {@link CreditCardPaymentMethod} by walletId
+ * Return undefined if not in list
+ */
+export const creditCardByIdSelector = createSelector(
+  [creditCardListSelector, (_: GlobalState, id: number) => id],
+  (potCreditCardList, id): CreditCardPaymentMethod | undefined =>
+    pot.getOrElse(
+      pot.map(potCreditCardList, creditCardList =>
+        creditCardList.find(cc => cc.idWallet === id)
+      ),
+      undefined
```

Comment:
```suggestion
    pot.toUndefined(
      pot.mapNullable(potCreditCardList, creditCardList =>
        creditCardList.find(cc => cc.idWallet === id)
      )
    )
```
totally optional. feel free to reject
it's a syntax sugar

Reply:
Changed in 786255294d95f570f2643617f26defd522c74d5e with: